src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -212,7 +212,23 @@ class DockerCreateContainer extends DockerExistingImage {
     @Input
     @Optional
     final Property<String> macAddress = project.objects.property(String)
-
+    
+    /**
+     * Set the IPC mode for the container
+     * "none"- Own private IPC namespace, with /dev/shm not mounted.
+     * "private" - 	Own private IPC namespace.
+     * "shareable" - Own private IPC namespace, with a possibility to share it with other containers.
+     * "container: <_name-or-ID_>" - Join another ("shareable") container’s IPC namespace.
+     * "host" - Use the host system’s IPC namespace.
```

Comment:
Can you please add a `@since` annotation with value 5.2.0?

Reply:
i think you got this one covered in the hostConfig refactor